src/app/phast/losses/operations/operations-form/operations-form.component.html

```diff
@@ -56,7 +56,16 @@
 </form>
 
 
-<app-operating-hours-modal *ngIf="showOperatingHoursModal" (emitClose)="closeOperatingHoursModal()"
-  (emitSave)="updateOperatingHours($event)" [width]="formWidth" [showMinutesSeconds]="false"
-  [operatingHours]="phast.operatingHours">
-</app-operating-hours-modal>
\ No newline at end of file
+<div bsModal #operatingCostsModal="bs-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="operatingCostsLabel"
```

Comment:
These changes shouldn't be part of this PR. Also if you haven't yet, switch to using VS code as your editor so you can have better management/visual on your git changes. See me if you have questions.